src/hooks/airdrop/stateHelpers/useAirdropTokensRefresh.ts

```diff
@@ -34,21 +36,21 @@ export async function handleRefreshAirdropTokens(airdropApiUrl: string) {
     const tokenHasExpired = tokenExpirationTime && tokenExpirationTime < expirationLimit;
     if (airdropTokens && tokenHasExpired) {
         try {
-            tokens = await fetchAirdropTokens(airdropApiUrl, airdropTokens);
+            tokens = await fetchAirdropTokens(airdropTokens);
         } catch (error) {
             console.error('Error refreshing airdrop tokens:', error);
         }
     }
+
     await setAirdropTokens(tokens);
 }
+
 export function useAirdropTokensRefresh() {
-    const backendInMemoryConfig = useAirdropStore((s) => s.backendInMemoryConfig);
     useEffect(() => {
-        if (!backendInMemoryConfig?.airdropApiUrl) return;
-        const interval = setInterval(
-            () => handleRefreshAirdropTokens(backendInMemoryConfig?.airdropApiUrl),
-            1000 * 60 * 60
-        );
-        return () => clearInterval(interval);
-    }, [backendInMemoryConfig?.airdropApiUrl]);
+        const REFRESH_INTERVAL = 1000 * 60 * 60; // an hour
+        const interval = setInterval(() => handleRefreshAirdropTokens(), REFRESH_INTERVAL);
+        return () => {
+            clearInterval(interval);
+        };
+    }, []);
```

Comment:
Any time there is an interval there's code smell. I don't think it needs to be addressed in this PR, but we need a wrapper around requests that checks the token expiry and refreshes the tokens only when expiry is imminent.

Reply:
@Krakaw ty for reviewing :D that was already there (😭) just moved it hehe, but yeah agree! i'll do that in a separate PR :D

Reply:
@Krakaw i ended up just doing it in this PR :D 
couldn't test with a real expiry because it's in two days' time, but tested with log in/out and restart with stored tokens